<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-16T00:09:25 IST" name="Suite" finished-at="2024-10-16T00:09:40 IST" duration-ms="14681">
    <groups>
    </groups>
    <test started-at="2024-10-16T00:09:25 IST" name="Test" finished-at="2024-10-16T00:09:40 IST" duration-ms="14681">
      <class name="com.mystore.testcases.TC_MyAccountPageTestDataDrivenTesting">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.mystore.testcases.TC_MyAccountPageTestDataDrivenTesting@5dda768f]" started-at="2024-10-16T00:09:25 IST" name="setup" finished-at="2024-10-16T00:09:34 IST" duration-ms="8476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLogin(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.mystore.testcases.TC_MyAccountPageTestDataDrivenTesting@5dda768f]" started-at="2024-10-16T00:09:35 IST" name="verifyLogin" data-provider="LoginDataProvider" finished-at="2024-10-16T00:09:38 IST" duration-ms="3150" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[prachi_456@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[prachi_456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Prachi gupta]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method signature="verifyLogin(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.mystore.testcases.TC_MyAccountPageTestDataDrivenTesting@5dda768f]" started-at="2024-10-16T00:09:38 IST" name="verifyLogin" data-provider="LoginDataProvider" finished-at="2024-10-16T00:09:40 IST" duration-ms="1741" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[prachi_456@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[prachi_456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Prachi dewen]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.mystore.testcases.TC_MyAccountPageTestDataDrivenTesting.verifyLogin(TC_MyAccountPageTestDataDrivenTesting.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
      </class> <!-- com.mystore.testcases.TC_MyAccountPageTestDataDrivenTesting -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
